Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -927,6 +927,16 @@ function vtkRenderWindowInteractor(publicAPI, model) {
       }
     }
   };
+
+  // Stop animating if the renderWindowInteractor is deleted.
+  const superDelete = publicAPI.delete;
+  publicAPI.delete = () => {
+    if (model.animationRequest) {
+      cancelAnimationFrame(model.animationRequest);
+      model.animationRequest = null;
+    }
```

Comment:
maybe it is safer to loop on `animationRequesters` and call `cancelAnimation` on them.
(you might want to disable rendering (`self.setEnableRender(false)`?) before doing so)

@jourdain @martinken  I'm surprised that `returnFromVRAnimation` does not call `handleAnimation()` if there are still some `animationRequesters` left (with maybe a preliminary `FrameTime=-1`)?

Reply:
The cancelAnimation is good here. We could clear the requestor map though. 

Kind of agree for your comment on `returnFromVRAnimation` but usually when we switch to VR we are not in an animation loop.